modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -120,9 +108,7 @@ def authorized_services
 
       def direct_deposit_update_access?
         user.authorize(:ppiu, :access_update?)
-      rescue => e
```

Comment:
The problem with logging this is that the error is almost always a timeout or backend service error. It's not helpful information.